src/builder/mod.rs

```diff
@@ -12,9 +12,16 @@ use crate::http::CacheHttp;
 #[cfg(feature = "http")]
 use crate::internal::prelude::*;
 
+/// Common trait for all HTTP request builders in this module.
+///
+/// Will have a [`serde::Serialize`]` + `[`serde::Deserialize`] bound in the future, because
+/// builders need to be serializable in order to send them to the Discord API and
+/// [`serde::Deserialize`] is useful to be able to store builders locally.
```

Comment:
```suggestion
/// Common trait for all HTTP request builders in this module.
```

I don't think the clarification here is necessary.